centreon/src/Core/Dashboard/Application/UseCase/ShareDashboard/ShareDashboard.php

```diff
@@ -0,0 +1,202 @@
+<?php
+
+/*
+ * Copyright 2005 - 2023 Centreon (https://www.centreon.com/)
```

Comment:
Applicable for all new files:
```suggestion
 * Copyright 2005 - 2024 Centreon (https://www.centreon.com/)
```

Reply:
Will try to update it but this change currently break the pipeline on modules, don't know if it is the case on web